app/main.py

```diff
@@ -1,6 +1,21 @@
+import requests
+import os
+
+API_KEY = os.getenv("API_KEY")
```

Comment:
Ensure that the `API_KEY` environment variable is set before running the script. Otherwise, the API request will fail.